package.json

```diff
@@ -25,7 +25,7 @@
     "prepack": "scripts/prepack.sh",
     "generate": "scripts/generate/run.js",
     "version": "scripts/release/version.sh",
-    "test": "hardhat test",
+    "test": "NODE_OPTIONS=\"--max-old-space-size=8192\" hardhat test",
```

Comment:
Can we remove that from the tests? 

My understanding is that it was only really needed for the coverage ... and we now have an elegant integration in coverage.sh.

Reply:
What are your thoughts on the current state?